src/sfizz/Debug.h

```diff
@@ -39,17 +39,31 @@
             ASSERTFALSE;   \
     } while (0)
 
+#define CHECKFALSE                                                              \
+    do {                                                                        \
+        std::cerr << "Check failed at " << __FILE__ << ":" << __LINE__ << '\n'; \
+    } while (0)
+
+#define CHECK(expression)  \
+    do {                   \
+        if (!(expression)) \
+            CHECKFALSE;    \
```

Comment:
Here if I may suggest, the expression may be stringified as `#expression` and displayed in the error.
The same applies to assert.

Reply:
:+1: